recipes/cyberpunk-theme/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xorg
```

Comment:
Is this really necessary for this recipe?

Reply:
It might have complained when I didn't add it but I might be mistaken, let me try to take it out.

Reply:
@ocefpaf looks like it does need it